README.md

```diff
@@ -0,0 +1,48 @@
+# GeoServer Workflow
+This script automates the process for preparing GeoTIFFs for use in WMS applications and adding them to GeoServer.
+
+This script is specific to Emory University's workflow and infrastructure, but could be adapted. Please feel free to open an issue here or contact <libsysdev-l@listserv.cc.emory.edu> if you have questions.
+
+Special thanks to Eric Willoughby at Georgia State University for working out the GDAL commands and consulting on GeoServer REST calls.
+
+## Dependencies
+You will need ruby and a few gems:
+
+<code>gem install net-sftp</code>
+
+<code>gem install httparty</code>
+
+<code>gem install nokogiri</code>
+
+## config.yaml
+Provided is a sample config file (<code>config.yaml.dst</code>). Rename that to <code>config.yaml</code> and fill in the various items.
+
+## What the script does
+* Collects all the tiff files in the configured directory
+* Runs each tiff though the following process one at a time
+	* Uses GDAL commands to prepare for WMS applications
+	* Finds the proper metadata file based on configured directory and matching file name. For example, if the script finds Sheet4.tif, it will look for Sheet4.xml
+	* Checks the metadata file for an ARK. If no ARK is found, it creates one and adds it to the metadata file
```

Comment:
Perhaps worth a brief description or link to explain an ARK? Also, do you need to document where the ARK is expected in the metadata?